src/thriveopedia/pages/ThriveopediaEvolutionaryTreePage.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Godot;
+
+/// <summary>
+///    Thriveopedia page displaying an evolutionary tree of all species in the current game.
+/// </summary>
+/// <remarks>
+///   <para>
+///     Note a lot of this functionality is duplicated from AutoEvoExploringTool.
```

Comment:
Again here, I think we should take steps to avoid code duplication to make sure the exploring tool and thriveopedia don't end up with different implementations of things in the long run.